src/PanZoom.js

```diff
@@ -240,15 +310,52 @@ class PanZoom extends React.Component<Props, State> {
   }
 
   onWheel = (e: WheelEvent) => {
```

Comment:
Wheel events are detected by the mouse scroll wheel and trackpad, so we get support for both by implementing an `onWheel` event handler. At the moment, we don't have a way to detect which device is sending events, more research would be required before knowing if it's possible.

Reply:
> we don't have a way to detect which device is sending events

As in: is it a mouse or a trackpad? 
How could we use that info?

Reply:
> As in: is it a mouse or a trackpad?

Yup yup 🎯

> How could we use that info?

We could tweak the pan speeds for the track pad and mouse wheel separately. We could speed up the mouse wheel while scrolling vertically and maybe slowdown the horizontal panning with the scroll wheel. The trackpad could also be configured to be slower or faster too.